src/mui/components/icon/IconByName.tsx

```diff
@@ -168,183 +158,196 @@ function rotateIcon(IconComponent: typeof SvgIcon, degrees: number) {
-    balanceOutlined: AccountBalanceOutlined,
-    batteryFlash: BatteryChargingFullOutlined,
-    calculateOutlined: CalculateOutlined,
-    card: CreditCard,
+    "entities.account": Person,
+    "entities.accounts": PublicOutlined,
+    "entities.job": PlaylistPlay,
+    "entities.material": Grain,
+    "entities.people": People,
+    "entities.project": Folder,
+    "entities.property": LooksOne,
+    "entities.subworkflow": Adjust,
+    "entities.task": Assignment,
+    "entities.team": Groups,
+    "entities.unit": RadioButtonUnchecked,
+    "entities.workflow": Lan,
+    "entities.material.nonPeriodic": Hub,
+
+    "unitType.assignment": SquareOutlined,
+    "unitType.condition": Code,
+    "unitType.convergence": RadioButtonUnchecked,
+    "unitType.execution": Circle,
+    "unitType.exit": Close,
+    "unitType.io": Storage,
+    "unitType.map": rotateIcon(CallSplit, 180),
+    "unitType.processing": RadioButtonChecked,
+    "unitType.reduce": rotateIcon(CallMerge, 180),
+    "unitType.subworkflow": Adjust,
+
+    "privacy.public": Public,
+    "privacy.withMe": Share,
+
+    "accounts.bank": AccountBalance,
+    "accounts.curators": LibraryBooks,
+    "accounts.enterprise": Language,
+    "accounts.externalPublic": Public,
+    "accounts.externalWithLink": Link,
+    "accounts.individual": AccountCircle,
+    "accounts.organization": GroupsRounded,
+    "accounts.personal": Face,
+    "accounts.public": People,
+
+    "billing.addBalance": CalculateOutlined,
+    "billing.card": CreditCard,
+    "billing.charge": Receipt,
+    "billing.credit": AttachMoney,
+    "billing.overview": AutoGraph,
+    "billing.payment": AttachMoney,
+    "billing.paymentFilled": Paid,
+    "billing.storageQuota": Memory,
+    "billing.totalBalance": AccountBalanceWalletOutlined,
+
+    "properties.bandGap": UnfoldLess,
+    "properties.energy": BatteryChargingFullOutlined,
+    "properties.energyProfile": SwapCalls,
+    "properties.fermiEnergy": FlashOn,
+    "properties.force": OpenWith,
+    "properties.pressure": SystemUpdateAlt,
+
+    "files.dropbox": SettingsSystemDaydream,
+    "files.externalUpload": CloudUploadOutlined,
+    "files.file": InsertDriveFile,
+    "files.fileText": Description,
+    "files.folder": FolderOutlined,
+
+    "pages.accounts": Groups,
+    "pages.billing": AttachMoney,
+    "pages.bio": Visibility,
+    "pages.compute": Settings,
+    "pages.computeLoad": DnsOutlined,
+    "pages.createJob": PlaylistAdd,
+    "pages.dashboard": Dashboard,
+    "pages.documentation": SupportOutlined,
+    "pages.help": HelpOutlined,
+    "pages.home": Home,
+    "pages.invite": PersonAddOutlined,
+    "pages.login": LoginOutlined,
+    "pages.logout": LogoutOutlined,
+    "pages.myAccounts": LanguageOutlined,
+    "pages.preferences": SettingsOutlined,
+    "pages.remoteDesktop": KeyboardOutlined,
+    "pages.serviceLevels": Layers,
+    "pages.support": SupportAgent,
+    "pages.terminal": Terminal,
+    "pages.jupyterNotebook": NorthEast,
```

Comment:
Nit: the northeast icon doesn't really represent the jupyter notebook, but rather the link to it (opened in new tab). Maybe `info.linkNewTab` would fit better here?

Reply:
changed to `info.externalLink`